word.md

```diff
@@ -304,6 +305,7 @@ Range of types
     rule #rangeAddress (            X ) => #range ( minUInt160      <= X <  pow160          )
     rule #rangeBytes   (   N ,      X ) => #range ( 0               <= X <  1 <<Byte N      )
     rule #rangeNonce   (   X          ) => #range ( 0               <= X < maxUInt64        )
+    rule #rangeSmall   (   X          ) => #range ( 0               <= X < 10               )
```

Comment:
Is this range only relevant for proof heuristics, or does it also have a significance for the semantics of EVM? (I'm thinking whether this is the right place to include it.)

Reply:
I've only seen that range being used in proofs. I also considered `lemmas.k`, but that file only contains simplifications.

Reply:
I think this is an appropriate place as well, it's next to the other range predicates.